### DISCUSSION TITLE
Create single stroke font with "open" contours useable for CNC-cutters

### DISCUSSION THREAD

**Discussion by stoltm** [fontforge/fontforge#5311] [Q&A]
Hi, 
I am trying to create a single stroke font - just like this one:
https://eng.fontke.com/font/76756248/download/

Therefore I try to create the "A" with as a path.
But acutally it closes itself like to be seen in the picture.

Is it possible to crate something like this in fontforge?
Do I just miss a setting or should my approach be a completely different?

![image](https://github.com/fontforge/fontforge/assets/36561796/d5b0906c-af4b-4c39-a473-ce92fd2054ac)

**Comment by skef** [comment]
This depends a lot on your tool chain.

The short answers are:

1. You can create glyphs with open contours in FontForge. If you accidentally close a contour you didn't intend to, there are a number of ways of eliminating one of the segments. One easy way is to use the cut tool to divide the segment and then select and delete the points.
2. There is somewhat obscure font format option for a "stroked" font, the sources of which will have open contours like you describe. I believe FontForge can generate these but offhand I don't know how. However, I believe most people using CnC fonts don't do this, they use a workflow like https://inkscape.org/~khemadeva/%E2%98%85inkscape-stroke-font-extensions . 

You might be able to use FontForge to create a single line font, save it as SVG, and then use those tools directly, but I'm not familiar with those workflows. someone on the https://sourceforge.net/p/fontforge/mailman/fontforge-users/ mailing list might be able to help.
